src/App.tsx

```diff
@@ -1,15 +1,40 @@
 import React from 'react';
-
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './Components/PostList';
+
+function capitalize(string: string) {
+  return `${string.charAt(0).toUpperCase() + string.slice(1)}.`;
+}
+
+const preparedPosts = posts.map(item => {
```

Comment:
```suggestion
const preparedPosts: Post[] = posts.map(item => {
```
В тебе є інтерфейс Post. Імпортуй його сюди і використовуй